.github/workflows/check.yml

```diff
@@ -48,3 +49,27 @@ jobs:
           --bazelrc=.github/workflows/ci.bazelrc \
           test \
           //...
+
+  clang-format:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: bazelbuild/setup-bazelisk@v2
+
+    - name: mount bazel cache
+      uses: actions/cache@v3
+      with:
+        path: "~/.cache/bazel"
+        key: bazel-clang-tidy
+
+    - name: install libtinfo5
+      # clang format links against this
+      run: sudo apt-get install -y libtinfo5
```

Comment:
Oh noes not hermetic! 
Can we have bazel create the symlink as described here?
https://stackoverflow.com/a/41517423/1720894

Could add it to `third_party/system_libs_linux_x86_64.BUILD.bazel`

Reply:
Sounds good. I'll look into this later unless you have a chance to do it before I do.

Reply:
It looks like `clang-format` that is pulled in is built linking to terminfo:
https://github.com/mun-lang/mun/issues/191#issuecomment-753402659
https://github.com/llvm/llvm-project/issues/53294

I took a look at how you use `system_libs` and you pass those targets as a dependency to `hipamd`. I'm not sure how we would do the same since `libtinfo` isn't passed as an explicit dependency.

Reply:
IIUC clang-format is pre-built, so we can't add deps to it, but I guess the right thing to do is if we can have the clang format aspect add a dep on the lib. But anyways I don't think it's worth spending time on unless you think it's fun.